app/js/views/filterView.js

```diff
@@ -0,0 +1,33 @@
+define([
+  'jquery',
+  'backbone',
+  'text!templates/filter_view.ejs',
+  'model/todo'
+], function ($, Backbone, FilterViewTemplate) {
+  return Backbone.View.extend({
+    events: {
+      'click #displayAll': 'displayAll',
+      'click #filter-done': 'filterDone',
+      'click #filter-not-done': 'filterNotDone'
+    },
+
+    render: function () {
+      var todoTemplate = ejs.render(FilterViewTemplate);
+
+      this.$el.append(todoTemplate);
+      return this;
+    },
+
+    displayAll: function () {
+      this.collection.each(function (todo) { todo.trigger('display'); });
+    },
+
+    filterDone: function () {
+      this.collection.each(function (todo) { todo.get('done') ? todo.trigger('display') : todo.trigger('hide'); });
```

Comment:
Not the best way to do it, here is why:

- add a couple of items
- mark one as done
- press the done filter
- add another item

Result:
The not done item is visible

Expected result
The not done item should not be visible.